.github/workflows/ci-frontend-e2e.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           cat .env.development >> .env.local
           # Use the localhost url for tests
           sed -i 's|^SENDY_API_URL=.*|SENDY_API_URL=http://localhost:3000|' .env.local
-          npm run build -- --no-lint
+          IS_CI=true npm run build -- --no-lint
```

Comment:
it looks like a `CI` env var is being set in the make task, and there's some stuff in the application using that variable so I'm assuming it's coming through. Is it possible to use that env var rather than creating a new one here?

Reply:
But IS... it CI?....

Good catch, testing.